src/components/Photon/photon.tsx

```diff
@@ -0,0 +1,216 @@
+        values.outputDir ? args.push(`-o`, `${values.outputDir}`) : undefined;
+        values.crawlDepth ? args.push(`-l`, `${values.crawlDepth}`) : undefined;
+        values.cookies ? args.push(`-c`, `${values.cookies}`) : undefined;
+        values.userAgent ? args.push(`-a`, `${values.userAgent}`) : undefined; // Changed -u to -a for userAgent
+
+        try {
+            const result = await CommandHelper.runCommandGetPidAndOutput(
+                "photon",
+                args,
+                handleProcessData,
+                handleProcessTermination
+            );
+
+            setPid(result.pid);
+            setOutput(result.output);
+        } catch (e: any) {
+            setOutput(e.message);
+            setLoading(false);
+        }
+    };
+
+    /**
+     * Clears the output state.
+     */
+    const clearOutput = useCallback(() => {
+        setOutput("");
+        setHasSaved(false);
+        setAllowSave(false);
+    }, [setOutput]);
+
+    return (
+        <RenderComponent
+            title={title}
+            description={description}
+            steps={steps}
+            tutorial={tutorial}
+            sourceLink={sourceLink}
+        >
+            {!loadingModal && (
+                <InstallationModal
+                    isOpen={opened}
+                    setOpened={setOpened}
+                    feature_description={description}
+                    dependencies={dependencies}
+                />
+            )}
+            <form onSubmit={form.onSubmit(onSubmit)}>
+                <Stack>
+                    {LoadingOverlayAndCancelButtonPkexec(loading, pid, handleProcessData, handleProcessTermination)}
+                    <TextInput
+                        label={"URL of the target"}
+                        placeholder={"Example: https://example.com"}
+                        required
+                        {...form.getInputProps("url")}
+                    />
+                    <TextInput
+                        label={"Output Directory"}
+                        placeholder={"Specify the output directory"}
+                        required
+                        {...form.getInputProps("outputDir")}
+                    />
+                    <TextInput
+                        label={"Crawl Depth"}
+                        placeholder={"Specify a number (Default = 2)"}
+                        {...form.getInputProps("crawlDepth")}
+                    />
+                    <TextInput
+                        label={"Cookies"}
+                        placeholder={"Specify cookies (if any)"}
+                        {...form.getInputProps("cookies")}
+                    />
+                    <TextInput
+                        label={"User Agent"}
+                        placeholder={"Specify a user agent (optional)"}
+                        {...form.getInputProps("userAgent")}
+                    />
+                    <Button type={"submit"}>Launch Crawler</Button>
```

Comment:
You should use Start {title}
```suggestion
                    <Button type={"submit"}>Start {title}</Button>
```

Reply:
thanks @Oliver2049 I done those thanks for that really appriciated.